modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,499 @@
+        port: 7564
+    - namedPort: "scrape"
+  - name: "allow-from-open-tenants" # rule2
+    action: "Allow"
+    from:
+    - namespaces: # open tenants
+        matchLabels:
+          tenant: open
+  - name: "pass-from-restricted-tenants" # rule3
+    action: "Pass"
+    from:
+    - namespaces: # restricted tenants
+        matchLabels:
+          tenant: restricted
+  - name: "default-deny" # rule4
+    action: "Deny"
+    from:
+    - namespaces: {}
+  egress:
+  - name: "allow-to-dns" # rule0
+    action: "Allow"
+    to:
+    - pods:
+        namespaceSelector:
+          matchlabels:
+            kubernetes.io/metadata.name: kube-system
+        podSelector:
+          matchlabels:
+            app: dns
+    ports:
+    - portNumber:
+        protocol: UDP
+        port: 5353
+  - name: "allow-to-kapi-server" # rule1
+    action: "Allow"
+    to:
+    - nodes:
+        matchExpressions:
+        - key: node-role.kubernetes.io/control-plane
+          operator: Exists
+    ports:
+    - portNumber:
+        protocol: TCP
+        port: 6443
+  - name: "allow-to-splunk" # rule2
+    action: "Allow"
+    to:
+    - namespaces:
+        matchlabels:
+          tenant: splunk
+    ports:
+    - portNumber:
+        protocol: TCP
+        port: 8991
+    - portNumber:
+        protocol: TCP
+        port: 8992
+  - name: "allow-to-open-tenants-and-intranet-and-worker-nodes" # rule3
+    action: "Allow"
+    to:
+    - nodes: # worker-nodes
+        matchExpressions:
+        - key: node-role.kubernetes.io/worker
+          operator: Exists
+    - networks: # intranet
+      - 172.30.0.0/30
+      - 10.0.54.0/19
+      - 10.0.56.38/32
+      - 10.0.69.0/24
+    - namespaces: # open tenants
+        matchLabels:
+          tenant: open
+  - name: "pass-to-restricted-tenants" # rule4
+    action: "Pass"
+    to:
+    - namespaces: # restricted tenants
+        matchLabels:
+          tenant: restricted
+  - name: "default-deny"
+    action: "Deny"
+    to:
+    - networks:
+      - 0.0.0.0/0
+----
+====
+
+.Procedure
+
+. List pods by running the following command:
++
+[source,terminal]
+----
+$ oc get po -n openshift-ovn-kubernetes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE
+ovnkube-control-plane-5c95487779-8k9fd   2/2     Running   0          33m
+ovnkube-control-plane-5c95487779-v2xn8   2/2     Running   0          33m
+ovnkube-node-524dt                       8/8     Running   0          33m
+ovnkube-node-gbwr9                       8/8     Running   0          24m
+ovnkube-node-h4fpx                       8/8     Running   0          33m
+ovnkube-node-j4hzw                       8/8     Running   0          24m
+ovnkube-node-wdhgv                       8/8     Running   0          33m
+ovnkube-node-wfncn                       8/8     Running   0          24m
+----
+
+. Optional: To list the pods with node information, run the following command:
```

Comment:
I'd remove step 1 and make this one non-optional, i.e. just ask the customer to always provide the output with node information.

Reason: Most of the times you troubleshoot something, you do it because you have problems on a node and you look for the ovnkube-node-XXXX pod of that node, so 99% of the times you want the pods with node information, so we can also keep the doc a bit shorter.

Reply:
done